delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/SupervisionRoutes.scala

```diff
@@ -13,34 +15,44 @@ import ch.epfl.bluebrain.nexus.delta.sdk.directives.DeltaDirectives.emit
 import ch.epfl.bluebrain.nexus.delta.sdk.directives.UriDirectives.baseUriPrefix
 import ch.epfl.bluebrain.nexus.delta.sdk.directives._
 import ch.epfl.bluebrain.nexus.delta.sdk.identities.Identities
+import ch.epfl.bluebrain.nexus.delta.sdk.marshalling.RdfMarshalling
 import ch.epfl.bluebrain.nexus.delta.sdk.model.BaseUri
 import ch.epfl.bluebrain.nexus.delta.sdk.permissions.Permissions.supervision
-import ch.epfl.bluebrain.nexus.delta.sourcing.stream.{SupervisedDescription, Supervisor}
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sourcing.stream.SupervisedDescription
 import io.circe.Encoder
 import io.circe.generic.semiauto.deriveEncoder
-import kamon.instrumentation.akka.http.TracingDirectives.operationName
+import io.circe.syntax.EncoderOps
 
 class SupervisionRoutes(
     identities: Identities,
     aclCheck: AclCheck,
-    supervised: IO[List[SupervisedDescription]]
+    supervised: IO[List[SupervisedDescription]],
+    projectsHealth: IO[List[ProjectRef]]
 )(implicit
     baseUri: BaseUri,
     cr: RemoteContextResolution,
     ordering: JsonKeyOrdering
-) extends AuthDirectives(identities, aclCheck) {
-
-  import baseUri.prefixSegment
+) extends AuthDirectives(identities, aclCheck)
+    with RdfMarshalling {
 
   def routes: Route =
     baseUriPrefix(baseUri.prefix) {
       pathPrefix("supervision") {
         extractCaller { implicit caller =>
           get {
-            operationName(s"$prefixSegment/supervision/projections") {
-              authorizeFor(AclAddress.Root, supervision.read).apply {
-                emit(supervised.map(SupervisionBundle))
-              }
+            authorizeFor(AclAddress.Root, supervision.read).apply {
+              concat(
+                (pathPrefix("projections") & pathEndOrSingleSlash) {
+                  emit(supervised.map(SupervisionBundle))
+                },
+                (pathPrefix("projects") & pathEndOrSingleSlash) {
+                  onSuccess(projectsHealth.unsafeToFuture()) { projects =>
```

Comment:
You could use the error channel like the other cases no ?

Reply:
I did it like this as I thought we said we didn't care about json-ld; if we raise an error then doesn't it have to have a JsonLdEncoder in order to be emitted with what we currently have?

Reply:
Oh, that is true.
I forgot that the error channel needed a json-ld encoder
Then it is ok like this if we have more endpoints like this, we can always invest time to change things later.

Reply:
Ok, I will just improve the json a bit.